testdata/default/cheats/Etch.t.sol

```diff
@@ -8,7 +8,7 @@ contract EtchTest is DSTest {
     Vm constant vm = Vm(HEVM_ADDRESS);
 
     function testEtch() public {
-        address target = address(10);
+        address target = address(11);
```

Comment:
Is this related to an new precompile in 0x10?

Reply:
yep, etch cheatcode has `ensure_not_precompile` and cancun added the new one